src/Popper/modifiers.js

```diff
@@ -1,16 +1,63 @@
-export const preventOverflow = {
-    name: 'preventOverflow',
-    options: {
-        altAxis: true,
-        rootBoundary: 'document',
-        boundary: document.body,
-    },
+import ResizeObserver from 'resize-observer-polyfill'
+
+const attachResizeObservers = ({
+    state: { elements },
+    options,
+    instance: { update },
+}) => {
+    const observers = Object.keys(options).reduce((acc, elementKey) => {
+        if (options[elementKey]) {
```

Comment:
Should this also check that `elements[elementKey]` exists?

Reply:
No, the popper.js state object will always contain `elements.popper` and `elements.reference`.